.github/workflows/ci.yml

```
@@ -18,6 +18,7 @@ jobs:
           - '1.6'
           - '1.8'
           - '1' # Leave this line unchanged. '1' will automatically expand to the latest stable 1.x release of Julia.
+          - '1.10.0-beta3'
```

```suggestion
          - 'beta'
```
I think this should run against the current beta release.